jenkins_pipelines/environments/salt-shaker/manager-salt-shaker-saltstack-leap154

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env groovy
+
+node('salt-shaker-tests') {
+    properties([
+        buildDiscarder(logRotator(numToKeepStr: '20', daysToKeepStr: '4', artifactNumToKeepStr: '3')),
+        disableConcurrentBuilds(),
+        pipelineTriggers([cron('H 0 * * *')]),
+        parameters([
+            choice(name: 'salt_flavor', choices: ['classic', 'bundle'], description: 'Run testsuite for classic Salt or Salt Bundle'),
+            booleanParam(name: 'run_unit_tests', defaultValue: true, description: 'Run the Salt unit tests'),
+            booleanParam(name: 'run_integration_tests', defaultValue: true, description: 'Run the Salt integration tests'),
+            booleanParam(name: 'run_functional_tests', defaultValue: true, description: 'Run the Salt functional tests'),
+            string(name: 'cucumber_ref', defaultValue: 'master', description: 'Testsuite Git reference (branch, tag...)'),
+            string(name: 'skip_list_url', defaultValue: 'https://raw.githubusercontent.com/openSUSE/salt-test-skiplist/main/skipped_tests.toml', description: 'URL to the skiplist.toml file to run Salt shaker'),
+            string(name: 'tf_file', defaultValue: 'susemanager-ci/terracumber_config/tf_files/salt-shaker/Salt-Shaker-Saltstack-Leap15.4.tf', description: 'Path to the tf file to be used'),
```

Comment:
I would rename the tf file to `Salt-Shaker-Saltstack-Leap154.tf`, to avoid the double file extension, as we do for others.